modules/kafka/kafka-grammar.ym

```diff
@@ -123,7 +124,20 @@ kafka_option
         ;
 
 kafka_properties
-        : kafka_property kafka_properties			{ $$ = g_list_prepend($2, $1); }
+	:
+	{
+		static CfgLexerKeyword empty_keywords[] = {{ CFG_KEYWORD_STOP }};
+		cfg_lexer_push_context(lexer, LL_CONTEXT_CONFIG, empty_keywords, "kafka_config_block");
+	}
+	kafka_properties_list
+	{
+		cfg_lexer_pop_context(lexer);
+		$$ = $2;
```

Comment:
`$2` does not seem to be a valid index here.

Reply:
Sorry, I was wrong, `kafka_properties_list` is going to be `$2` here due to the first block, in which we create a context.